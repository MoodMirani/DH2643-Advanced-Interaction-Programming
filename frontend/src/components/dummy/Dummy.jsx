import React from "react";

export const Dummy = ({shouldRenderText, text}) => {

    return shouldRenderText && <div>{text}</div>

}