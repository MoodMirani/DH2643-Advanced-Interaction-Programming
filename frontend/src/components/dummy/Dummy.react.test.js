import React from "react";
import jestRenderer from 'react-test-renderer';
import { Dummy } from "./Dummy";

test("The component should render our test if the prop is true", () => {
    const component = jestRenderer.create(
        <Dummy shouldRenderText={true} text="Hello World" />
    )
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})
