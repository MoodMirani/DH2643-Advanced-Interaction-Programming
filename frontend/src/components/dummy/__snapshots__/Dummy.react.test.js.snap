// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component should render our test if the prop is true 1`] = `
<div>
  Hello World
</div>
`;
